src/middlewares/auth/verifyRole.js

```diff
@@ -0,0 +1,14 @@
+const verifyRole = (...allowedRoles) => {
+  return (req, res, next) => {
+    if (!req?.headers.role) return res.boom.unauthorized();
+
+    const rolesArray = [...allowedRoles];
```

Comment:
This action is useless. You're spreading array to another array. You can just write if(!allowedRoles.incudes(req.userInfo.role) return res.boom.unauthorized()

Reply:
Fixed. Thanks!